04-GitRStudio.Rmd

```diff
@@ -91,9 +102,14 @@ knitr::include_graphics("images/Git15.PNG")
 
 Now you are ready to start interacting with GitHub from RStudio.
 
-Here are some very basic instructions to get you started. As you are going about your regular coding business working in an `R Project`, the .git will be keeping track of the changes you make to the files in this repo. To move these changes from your local repo back to the `origin` on the cloud, there are four simple steps to follow: `stage`, `commit`, `pull`, and `push`.
+Here are some very basic instructions to get you started. As you are going about your regular coding business working in an `R Project`, Git will be keeping track of the changes you make to the files in this repo. To move these changes from your local repo back to the `origin` on the cloud, there are four simple steps to follow: `stage`, `commit`, `pull`, and `push`.
 
-Not every change is by default included in every commit. To select the changes you wish you commit, you `stage` them (think "add them to a staging area"). In the `Git` tab in RStudio, you will see check boxes under the word `staged`. This allows you to select which changes you are going to commit to your local repo. Having such fine grained control over which files (and even which *lines* are committed) is certainly handy, but many people typically just stage all changes each commit. If you have many modified files you can easily stage them all by selecting them all using `CTRL A` then click the checkbox on any one of the files.
```

Comment:
I think it's important to point out how to stage parts of files, just in case :wink: